crates/ln-dlc-node/src/node/wallet.rs

```diff
@@ -28,32 +29,21 @@ where
         self.wallet.get_seed_phrase()
     }
 
-    pub fn sync(&self) -> Result<()> {
-        let confirmables = vec![
-            &*self.channel_manager as &dyn Confirm,
-            &*self.chain_monitor as &dyn Confirm,
-        ];
-
-        self.wallet
-            .inner()
-            .sync(confirmables)
-            .context("Failed to sync wallet")
+    pub fn wallet(&self) -> Arc<ldk_node_wallet::Wallet<sled::Tree>> {
+        self.wallet.inner()
     }
 
     pub fn get_new_address(&self) -> Result<Address> {
```

Comment:
🤔  It's confusing that this method is called `get_new_address` but is returning `get_last_used_address`. Is this intentional?